src/blockchain/qtum.ts

```diff
@@ -1,16 +1,18 @@
-import {get, post} from '../connector/tatum';
-import {QtumIGetInfo, QtumIRawTransactionInfo, QtumIRawTransactions, QtumISendRawTxResult, QtumIUTXO} from '../model';
-import {QtumBlock} from '../model/response/qtum/QtumBlock';
 
-export const getQtumUTXOs = async (address: string): Promise<QtumIUTXO> => get(`/v3/qtum/utxo/${address}`);
-export const getQtumTransaction = async (id: string): Promise<QtumIRawTransactionInfo> => get(`/v3/qtum/transaction/${id}`);
-export const getQtumTransactions = async (address: string, pageNum: number): Promise<QtumIRawTransactions> => get(`/v3/qtum/transactions/address/${address}/${pageNum}`);
+import { get, post } from '../connector/tatum'
+import { QtumIRawTransactions, QtumIRawTransactionInfo, QtumISendRawTxResult } from "src/model/response/qtum/QtumTx"
+import { QtumIGetInfo } from 'src/model/response/qtum/QtumInfo';
+import { QtumIUTXO } from 'src/model/response/qtum/QtumUTXO';
+import { QtumBlock } from '../model/response/qtum/QtumBlock';
 
+export const getQtumUTXOs = async (address: string): Promise<QtumIUTXO> => get(`/v3/qtum/utxo/${address}`);
+export const getQtumTransaction = async (id: string): Promise<QtumIRawTransactionInfo> => get(`/v3/qtum/transaction/${id}`)
+export const getQtumTransactions = async (address: string, pageSize: number, offset:number): Promise<QtumIRawTransactions> => get(`/v3/qtum/transactions/address/${address}/?pageSize=${pageSize}&offset=${offset}`)
```

Comment:
make pageSize default to 50 and offset to 0 pls

Reply:
done